File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/Fortran/solution.f
INTEGER :: num, n

READ *, num
n = num

IF (MOD(n, 2)==1) THEN
    WRITE(*,'(a)') "Weird"
ELSE IF (n >= 6 .AND. n <= 20) THEN
    WRITE(*,'(a)') "Weird"
ELSE
    WRITE(*,'(a)') "Not Weird"
END IF

END

File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/Clojure/solution.clj
(use '[clojure.string :only (split triml)])

(
        let [
          N_t (read-line) 
          N (Integer/parseInt N_t) 
        ]
        
        (if (== (mod N 2) 1) 
            (println "Weird") 
            (if (and (>= N 6) (<= N 20)) 
                (println "Weird") (println "Not Weird")))

)
